qa/tasks/cephfs/filesystem.py

```diff
@@ -1273,6 +1273,10 @@ def read_cache(self, path, depth=None, rank=None):
 
         return result
 
+    def restart_mds(self):
+        self.fail()
+        self.set_joinable()
```

Comment:
You can use `rank_restart()` or `mds_fail_restart()` to restart MDS

Reply:
Done. IIUC, `restart_mds()` and the chosen `mds_fail_restart()` function similar for this test.

Reply:
What Chris had was actually the right thing. We do not want to restart the daemons.